Makefile

```diff
@@ -1,9 +1,25 @@
+TOPDIR := $(shell pwd)
+
 CC = gcc
-CFLAGS = -I./include -I/usr/include/keapi -Wall -Wextra -pedantic -g -O0
-LDFLAGS = -lkeapi
+CFLAGS ?= -Wall -Wextra -pedantic -g -O0
+LDFLAGS ?= -lkeapi
+
+PKG_CONFIG ?= pkg-config
+INSTALL ?= install
+
+# install directories
+PREFIX ?= /usr
+SBINDIR ?= $(PREFIX)/bin
```

Comment:
The value of  SBINDIR should be $(PREFIX)/sbin

Reply:
yes for sure .. I only changed the variable name and forgot the value ;-(